app/forms/schools/coaches/update_form.rb

```diff
@@ -50,13 +50,26 @@ def user_params
       end
 
       def faculty_params
-        { connect_link: connect_link, public: public, exited: exited }
+        { connect_link: connect_link, public: public }
       end
 
       def faculty
         @faculty ||= school.faculty.find_by(id: id)
       end
 
+      def mark_faculty_as_exited(exited)
+        should_exit = exited == "true" ? true : false
+
+        if !faculty.exited? && should_exit
+          faculty.exited = true
+          faculty.save!
+          clear_faculty_enrollments
+        elsif faculty.exited? && !should_exit
+          faculty.exited = false # unarchive
+          faculty.save!
+        end
+      end
```

Comment:
You can revert all the changes in this file; Marking as exited worked before, so why update it here?

Reply:
Yeah, complex logic for nothing 😂.